README.md

```diff
@@ -115,6 +115,62 @@ Follow the instructions [here](https://www.mediawiki.org/wiki/OAuth/Owner-only_c
 
 For an example on how to request a edit token and make an authenticated request using OAuth see the [example python script](example/oauth/main.py).
 
+## Property lookup by name
```

Comment:
I think both “property lookup by name” and “statement reconciliation” should be subsections of the minimal input format, since we don’t support them for the full input format.

Reply:
Good point.